x-pack/plugins/security_solution/public/management/pages/endpoint_hosts/mocks.ts

```diff
@@ -72,6 +72,30 @@ export const endpointMetadataHttpMocks = httpHandlerMockFactory<EndpointMetadata
         };
       },
     },
+    {
+      id: 'metadataList',
```

Comment:
you need to change the ID here and define it in the type as well. Unless you are remove the one above? This `id` is gets converted the property key of `mockedApis.responseProviders[id_value_here]`, so defining it twice will actually override the one above and we would not be able to manipulate it.

(and now I'm also wondering if I can get TS to actually detect this 🤔 )

Reply:
didn't see that one, thanks for pointing that out. I do plan on removing it, would it make more sense to do it in the removal PR? https://github.com/elastic/kibana/pull/119401.

Reply:
sure, that's fine 👍